src/main/java/com/example/spring/services/UserServiceImpl.java

```diff
@@ -23,6 +28,10 @@ public UserResponseDto register(UserRegistrationRequestDto registrationRequestDt
                     + " already exists");
         }
         User user = userMapper.toModel(registrationRequestDto);
+        user.setPassword(passwordEncoder.encode(user.getPassword()));
+        System.out.println(user);
+        user.setRoles(new HashSet<>());
+        user.getRoles().add(roleRepository.findById(2).orElse(null));
```

Comment:
```suggestion
        user.setRoles(Set.of(userRole));
```

userRole - find Role user from repo